lib/Command/AddCommand.php

```diff
@@ -85,14 +81,20 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 			return 1;
 		}
 
+		$email = $input->getArgument('email');
+		if ($this->config->useHashedEmailAsUserID()) {
+			$email = strtolower($email);
```

Comment:
Should that be mb_strtolower like in UserBackend?

Reply:
mb is not allowed in user ids nor in emails, so not sure it's required?